editor/reducer.js

```diff
@@ -435,6 +402,7 @@ export function showInsertionPoint( state = false, action ) {
  * @return {string}                        Updated state
  */
 export function preferences( state = STORE_DEFAULTS.preferences, action ) {
+	const maxRecent = 8;
```

Comment:
Minor: Could this be defined as a module constant above: `const MAX_RECENT_BLOCKS = 8;` (we use this unofficial guideline elsewhere for constants)